### PR TITLE
Updating error messages with expected default formats

### PR DESCRIPTION
**Pull request by CrucisGamma**
# Description of the Change

Expand the error message to include the required data format for data input that don't match various elements in the code 

### Alternate Designs

Modifying for everything including floats/integers was considered but I think most people know this already?

### Why Should This Be In Core?

As a user, would be helpful to know what internal format will be used

### Benefits

As a user, would be helpful to know what internal format will be used

### Possible Drawbacks

Will need to be maintained if the internal format does end up changing

### Verification Process

- How did you verify that all new functionality works as expected?
Ran and tested this manually (for example, ingesting a new .csv with an incorrect datetime format)
- How did you verify that all changed functionality works as expected?
This is only modifying the existing error message
- How did you verify that the change has not introduced any regressions?
Ran existing tests

### CODE REVIEW

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportController.java [hunk 1]
```diff
@@ -170,7 +171,12 @@ public void setClearManuallyAdded(final boolean b) {
     }
 
     public void setDestination(final ImportDestination<?> destination) {
-        this.currentDestination = destination;
+        if (destination != null) {
+            this.currentDestination = destination;
+        }
+        if (this.currentDestination == null) {
+            return;
```
Comment (antares1470) [inline CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportController.java:178]: sonar will probably flag this return as unnecessary

Comment (arcturus2) [reply CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportController.java:178]: Hi @CrucisGamma so @antares1470 has been smashing through SQ rules so any assistance to keep code smells down is greatly appreciated. What i'll do is merge it into a temporary branch  to see what SQ has to say. SQ can't run on your fork.

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/RunPane.java [hunk 2]
```diff
@@ -186,6 +191,19 @@ public RunPane(final ImportController importController) {
 
         attributePane.addRow(0, sourceVertexScrollPane, destinationVertexScrollPane, transactionScrollPane);
 
+        attributePane.setOnKeyPressed(event -> {
+            final KeyCode c = event.getCode();
```
Comment (arcturus2) [inline CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/RunPane.java:195]: :+1:

**Comment by arcturus2** [comment]
Interesting @antares1470 so SQ didn't complain about the empty return  - I expected it would too. See #401. Going to merge #401.

**Pull request merged**